### ISSUE TITLE
Provide simple frontend to trigger and view square AVS

### ISSUE THREAD

**Issue by ethanfrey** [Lay3rLabs/example-avs-ui#3]
Depends on #2 
Depends on https://github.com/Lay3rLabs/example-avs-square/issues/2

The focus here is the AVS user (simulating what a calling contract sees), not the AVS developer. They can use CLI tools to deploy (for now).

This provides a route like `/avs/square/{address}` that:

* Shows all open and closed tasks on the AVS task queue (from newest to oldest)
* Allows creating a new task on the queue (not arbitrary JSON, but accept one number from the user and format into valid JSON)
* Submits and signs task with the wallet
* Provides "real-time" updates of the view, so when a task is completed, it shows up as completed within 1-2s.

This should be a basic, but clean and attractive design. Easy to use and easy to modify the code. Please add nice comments in the code for anything non-intuitive

For a demo, also add a redirect of the `/` route to a fixed task queue, so going to an app immediately shows the desired AVS. This should be trivial to update to point it to a different address.
